extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeMixPlaylistExtractor.java

```diff
@@ -211,14 +217,19 @@ public InfoItemsPage<StreamInfoItem> getPage(final Page page) throws IOException
         }
 
         final StreamInfoItemsCollector collector = new StreamInfoItemsCollector(getServiceId());
+
         final Map<String, List<String>> headers = new HashMap<>();
         addClientInfoHeaders(headers);
+        //noinspection ArraysAsListWithZeroOrOneArgument
+        headers.put("Cookie", Arrays.asList(page.getCookies().get(COOKIE_NAME)));
```

Comment:
Why not `Collections.singletonList()`? If there is a reason, please add a comment

Reply:
I will add a comment (that's to allow adding cookies by extractor clients such as the reCAPTCHA one).